imageroot/actions/get-configuration/20read

```diff
@@ -23,14 +24,27 @@ config["lets_encrypt"] = os.getenv("TRAEFIK_LETS_ENCRYPT") == "True"
 
 config["mail_server"] = os.getenv("MAIL_SERVER","")
 config["plugins"] = os.getenv("ROUNDCUBEMAIL_PLUGINS_NETHSERVER","")
-config["upload_max_filesize"] = int((os.getenv("ROUNDCUBEMAIL_UPLOAD_MAX_FILESIZE","5").replace('M','')))
-config["imap_port"] = int(os.getenv("ROUNDCUBEMAIL_DEFAULT_PORT","143"))
-config["smtp_port"] = int(os.getenv("ROUNDCUBEMAIL_SMTP_PORT","587"))
-
-config["encrypt_imap"] = os.getenv("ENCRYPT_IMAP",'starttls')
-config["encrypt_smtp"] = os.getenv("ENCRYPT_SMTP",'starttls')
-config["tls_verify_imap"] = os.getenv("TLS_VERIFY_IMAP") == "True"
-config["tls_verify_smtp"] = os.getenv("TLS_VERIFY_SMTP") == "True"
+config["upload_max_filesize"] = os.getenv("ROUNDCUBEMAIL_UPLOAD_MAX_FILESIZE","10").replace('M','')
+
+rdb = agent.redis_connect(host='127.0.0.1')
+modules=[]
+
+# we query about all mail server
+for key in agent.list_service_providers(rdb,'imap','tcp'):
+
+    # mail_hostname could be optional, use user_domain instead
+    mail_hostname = key['user_domain'] if 'mail_hostname' not in key else key['mail_hostname']
```

Comment:
```suggestion
    mail_hostname = key.get('mail_hostname') or key.get('user_domain')
```

Reply:
`mail_hostname = key.get('mail_hostname', key.get('user_domain'))`

does it work :D

Reply:
@davidep, what about to discovery when the mail server hasb been moved, I saw that srv discovery has an event  for this purpose, what do you think ?